ci/test_python.sh

```diff
@@ -179,6 +179,7 @@ if [[ "${RAPIDS_CUDA_VERSION}" == "11.8.0" ]]; then
       --channel "${PYTHON_CHANNEL}" \
       libcugraph \
       pylibcugraph \
+      pylibcugraphops \
```

Comment:
```suggestion
```

This list is reserved for only packages that are built as part of the `cugraph` repository workflow.

If this is a run time dependency, it should be added to the relevant conda recipe and the relevant [dependencies.yaml](https://github.com/rapidsai/cugraph/blob/branch-23.06/dependencies.yaml) section.

If it's strictly a test dependency, it should only be added to the relevant [dependencies.yaml](https://github.com/rapidsai/cugraph/blob/branch-23.06/dependencies.yaml) section so it can be used in the conda environment that's [generated earlier in this script](https://github.com/rapidsai/cugraph/blob/5bfa7c5a1a6c540f580649c02a1982ea65e32741/ci/test_python.sh#L8-L14).

Reply:
@ajschmidt8 I would still need to add `pylibcugraphops` to this file to enable relevant tests in `cugraph_pyg`.

I understand that this code block:
https://github.com/rapidsai/cugraph/blob/branch-23.06/ci/test_python.sh#L31-L38
is reserved for `cugraph` packages. 

The one that I modified is in `test_cugraph_pyg`, which is intended for including additional test dependencies (like pytorch) without polluting the `dependencies.yaml`. @alexbarghi-nv, correct me if I am wrong, since I am not involved in the original discussion of this design.

If you look at the code block for `test_cugraph_dgl`, `pylibcugraphops` is already there.